### ISSUE TITLE
Running `-Qi` on a `-deb` package doesn't return size

### ISSUE THREAD

**Issue by Henryws** [pacstall/pacstall#250]
**Describe the bug**
When installing a deb package (balena-etcher-electron-deb), running `pacstall -Qi balena-etcher-electron-deb` doesn't return the size of the package (because it's not symlinked in `/usr/src/pacstall`)

**To Reproduce**
Install pacstall, then install `balena-etcher-electron-deb`, or any other `-deb` package, run `pacstall -Qi <pkg>`, and see it doesn't return the size

**Expected behavior**
For it to return the size of the package, or, not at all

**Desktop (please complete the following information):**

- OS + version: Debian 11
- Pacstall Version + branch: 1.6 develop

**Comment by wizard-28** [comment]
We can use `apt-cache show <pkg> | grep Installed-Size | cut -d: -f 2`. Can you suggest a slim `-deb` pacstall package to test this on?

**Comment by wizard-28** [comment]
It returns values without unit...
Example:
```bash
 apt-cache show sway-backgrounds | grep Installed-Size | cut -d:
-f 2
 4930
```

**Comment by wizard-28** [comment]
Summoning `-deb` pensioner @NarutoXY too.

**Comment by NarutoXY** [comment]
well we can do another case in the `-Qi` section to show `-deb` package size using `apt`. There is also a case to be made for the problems with not having a united function and this will probably be solved by 2.0
